include/hc_am.hpp

```diff
@@ -155,5 +155,9 @@ void am_memtracker_print();
 void am_memtracker_sizeinfo(const hc::accelerator &acc, size_t *deviceMemSize, size_t *hostMemSize, size_t *userMemSize);
 
 
+am_status_t am_memtracker_host_memory_lock(hc::accelerator &acc, void *ptr, size_t size);
```

Comment:
@adityaatluri As these two new functions are public APIs and would supposedly be invoked by user applications, could you help provide some comments for them?